fastlane/Fastfile

```diff
@@ -24,29 +24,34 @@ platform :android do
       else
         raise "Unknown flavor #{options[:flavor]}"
       end
+
     version_codes = google_play_track_version_codes(
       package_name: package_name,
       track: "internal",
     )
     last_version_code = version_codes.max
     next_version_code = last_version_code + 1
```

Comment:
Fetching the existing version codes from Google Play requires GCP authentication, which won't be possible for PRs from external contributors. I think if the build type is debug we can just set `next_version_code = 1`.